report/sql_tasks/tasks/report/create_from_scratch/03_public/01_materialized_views/02_organization_activity/01_create_view.sql

```diff
@@ -54,8 +57,8 @@ CREATE MATERIALIZED VIEW organization_activity AS (
             organizations.public_id = companies.lms_organization_id
         WHERE
             organizations.id = organization_id
-            AND deals.services_start <= calendar_date
-            AND deals.services_end >= calendar_date
+            -- Check for overlapping date ranges
+            AND DATERANGE(deals.services_start, deals.services_end, '[)') && DATERANGE(start_date, end_date, '[)')
```

Comment:
Oh so this is constructing a range type. Clever design from (postgres? sql?) on that last argument.

Reply:
This is similar to mathematical notation I think to denote inclusive `[` or exclusive `)`.